0-ad-history-mod-package/simulation/templates/structures/han/civil_centre.xml

```diff
@@ -10,6 +10,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Guān Shǔ</SpecificName>
+    <History>
+         The Han empire was absolutely governed by an emperor who would pass on the throne to his eldest son. He was the supreme lawgiver, chief commander of the military, and designator of senior government officials. However, the emperor was assisted by a complex government system. It consisted of numerous layers and institutions and was meant to simplify political procedures without letting anyone (except the emperor) amass power. The system was structured as follows (simplified):
+         At the top were three officials (called The Three Excellencies), each responsible for a different domain: The Chancellor stood above the population and the lands and was responsible for the state's finances. The Grand Commandant held command over the empire's military. The Imperial Secretary had the duty of overseeing and monitoring the government and its officials.
+         Below these were nine ministries, operating on different matters such as finance, justice, and ceremonies. Government service encompassed twelve total levels; the officials were classified and could rank up based on their capabilities and merit, not their ancestry, which also helped prevent corruption and greed for power.
```

Comment:
```suggestion
         Below these were nine ministries, which operated on different matters such as finance, justice, and ceremonies. Government service encompassed twelve total levels; the officials were classified and could rank up based on their capabilities and merit, not their ancestry, which also helped prevent corruption.
```

This paragraph is pretty good, except from what I read there was still plenty of corruption in the Han government, so I think that aside needs to be reworded into something less solid. (This was supposed to prevent corruption)